test/functional/apps/dashboard/group4/dashboard_save.ts

```diff
@@ -125,6 +126,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('Does not show dashboard save modal when on quick save', async function () {
+      await esArchiver.loadIfNeeded('test/functional/fixtures/es_archiver/logstash_functional');
```

Comment:
ℹ️ I found out that on this test, there were no es data loaded, and with the change done on this PR, we need es data in order to have access to the viz editor. So I am loading them on this test.